perl/README.md

```diff
@@ -0,0 +1,1028 @@
+
+3. **Insertions and Deletions**: For transcript-level annotations like `refSeq.*` (refSeq transcript annotations), `nearest.*`, and `nearestTss.*` (nearest gene by transcript boundaries and by distance to the transcription start site respectively), insertions and deletions affecting multiple nucleotides are separated by pipes (`|`). This allows reporting a transcript consequence per disrupted base.
+
+4. **Reserved Delimiters**: The reserved delimiters described in points 1-3 (`;`, `/`, and `|`) will be stripped and replaced with a comma if found in source inputs to the Bystro Annotation Database.
+
+## What Information Can Bystro Annotator Output?
+
+Bystro Annotator is a general-purpose data curation and labeling engine for genetic data, and has no restrictions on the types of annotations/feature labels it can output. Theoretically it can even support binary data, such as images.
+
+## Variant Representations
+
+Bystro's variant representation deviates slightly from the standard VCF format in the name of simplicity. In particular, it drops the rule that the alternate allele must be ACTG. Dropping this single restriction allows us to represent deletions as occuring at the actual first deleted base, rather than the base before, as in the VCF format. This has a number of knock on benefits:
+
+- The `inputRef` (reference base) in Bystro's annotation outputs is always exactly 1 base long
+
+- The `pos' (position) in Bystro's annotation outputs is always the first affected base, except in the case of insertions, where it is the base before the insertion, since the insertion by definition is between two reference bases
+
+- It is possible to represent all multiallelic site using a single reference base, a single position, and a list of alleles
+
+The Bystro Genotype Dosage Matrix, is a columnar dataset, generated for every collection of VCFs submitted. Its first column is the `locus`, which is `chr:pos:ref:alt`. Every column after that is labeled by the sample name, and contains a -1 for missing genotypes, 0 for reference, 1 for a single alternate allele, 2 for 2 alternate alleles (homozygous in a diploid organism), and so on. It can be used for many things, such as to calculate polygenic risk scores.
+
+### Comparing the VCF and Bystro Variant Representations
+
+Below we'll demonstrate the Bystro Annotation and Dosage Matrix outputs, using some examples. You may run these examples, if you have Bystro installed, or at least the Bystro VCF Preprocessor. The Bystro VCF Preprocessor, which is a tool that takes a VCF file and outputs a Bystro-annotator compatible TSV file, which is then used by the Perl Bystro Annotator to add annotations to the VCF file from the Bystro Annotation Database. You can install it by running `go install github.com/bystrogenomics/bystro-vcf@2.2.3`.
+
+Please note that we are not showing the full Bystro Annotator outputs, which are far too extensive to easily display here.
```

Comment:
Recommend to just give numbers (just used example numbers below) so that it doesn't sound like it's impossible to look at or think about, but it is large enough that it's clearly impractical to show everything.

```suggestion
Please note that we are not showing the full Bystro Annotator outputs, just the first 20 fields out of the 150 fields that we annotate
```

Reply:
done